### ISSUE TITLE
Body not showing in the API documentation on AWS

### ISSUE THREAD

**Issue by PieterBogemans** [swagger-api/swagger-ui#4554]
So recently I updated an endpoint to my API. I made it so the endpoint uses a body instead of parameters because a client requested it this way. 

It works perfectly fine through something like Postman but the swagger api documentation doesn't show the body after pressing "try out". Locally this works perfectly fine but on AWS I get this: 

![image](https://user-images.githubusercontent.com/25767072/40003327-58ba67ce-5793-11e8-881c-cec07c94af9e.png)

Local:

![image](https://user-images.githubusercontent.com/25767072/40003394-7fee7560-5793-11e8-9de4-a162811bcfeb.png)

I doubt that it has anything to do with my implementation of the controller or the service since it works locally. What could possibly trigger this behavior on AWS?

**Comment by shockey** [comment]
Interesting!

@PieterBogemans, can you provide a definition and server URL that helps us reproduce this?

**Comment by PieterBogemans** [comment]
Some code of the endpoints (not that I expect anything to be wrong with this):

![image](https://user-images.githubusercontent.com/25767072/40169673-cf5f678c-59c6-11e8-8b93-22e452a2db52.png)

![image](https://user-images.githubusercontent.com/25767072/40169681-d4e6b2e6-59c6-11e8-97d0-670051384b52.png)

Unfortunately I'm not allowed to actually show you the repository. Do you have an idea what could cause this issue? I had a difference in version of node modules in mind (AWS vs local) but I tried to have fixed dependencies, which didn't work.

**Comment by shockey** [comment]
@PieterBogemans, hmm... is the request actually coming back in both environments? Are there any browser console errors in either environment?

**Comment by shockey** [comment]
@PieterBogemans I see.

Can you record a screencast of what's happening? I get the impression that I'm missing something from just looking at the screenshots (you mention differences in what happens before and after pressing Try-It-Out that sound hard to capture in a screenshot).

Also - have you configured your AWS deployment to serve CORS headers?

**Comment by shockey** [comment]
@PieterBogemans, yeah, it definitely sounds like something changed on our end that broke your implementation.

It's hard to say what changed, because (a) I don't know what version you were on before, there are 68 versions of Swagger-UI that look like the version in your screenshots and (b) I haven't seen your Swagger document - it's possible that there are errors, or you're doing something esoteric that broke in a recent release.

Sharing a document that helps me reproduce this (or even better, the one you're using) would go a long way towards getting this fixed :)

**Comment by PieterBogemans** [comment]
After playing around with the versions locally and it now being pushed online I confirmed that these versions work. I still don't really know what caused this issue but these are the versions I'm currently using: 

"@nestjs/common": "4.5.9",
"@nestjs/core": "4.5.10",
"@nestjs/microservices": "4.5.8",
"@nestjs/swagger": "1.2.2",
"@nestjs/testing": "4.5.5",
"@nestjs/websockets": "4.5.8",


It's weird because I just deleted all node_modules at some point and then reinstalled them and now it works. So I hardcoded the versions which I assume are the latest. 
Let me know if there's still something you'd like to know about this issue.
